openmls/src/ciphersuite/aead.rs

```diff
@@ -136,17 +148,14 @@ impl AeadNonce {
 }
 
 #[cfg(test)]
-pub(crate) fn aead_key_gen(
-    alg: openmls_traits::types::AeadType,
-    rng: &impl OpenMlsRand,
-) -> Vec<u8> {
-    match alg {
+pub(crate) fn aead_key_gen(ciphersuite: Ciphersuite, rng: &impl OpenMlsRand) -> AeadValue {
```

Comment:
Rollback to the previous API which was better
```suggestion
pub(crate) fn aead_key_gen(alg: openmls_traits::types::AeadType, rng: &impl OpenMlsRand) -> AeadValue {
```